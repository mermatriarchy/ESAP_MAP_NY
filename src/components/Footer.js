import React from 'react';
import {Container, Row, Col} from 'react-bootstrap/';

export default function Footer() {
  return (
    <Container as={`footer`}>
      <Row>
        <Col md={6}>
          <p>Copyright 2019-2020, H. Adonis Miller</p>
          <p>This is a community-focused project not intended for use by police, military personnel or any adjacent third party affiliates or contractors; it is not intended to aid in surveillence of our communities in any capacity.</p>
          <p><a href="">Source</a></p>
          <p><a href="">Report an Issue</a></p>
        </Col>
      </Row>
    </Container>
  );
}
