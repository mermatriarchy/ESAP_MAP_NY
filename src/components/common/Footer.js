import React from 'react';
import {Container, Row, Col} from 'react-bootstrap/';

export default function Footer() {
  return (
    <Container as={`footer`} className="bg-dark" fluid={true}>
      <Row>
        <Col md={6}>
          <p className="text-white">
            Copyright 2019-2020, H. Adonis Miller
            <br/>
            <span className="font-weight-light">This is a community-focused project not intended for use by police, military personnel or any adjacent third party affiliates or contractors; it is not intended to aid in surveillence of our communities in any capacity.</span></p>
        </Col>
      </Row>
      <Row>
        <Col>
          <dl>
            <dd><a href="" className="text-info">Source</a></dd>
            <dd><a href="" className="text-info">Report a technical issue</a></dd>
          </dl>
        </Col>
      </Row>
    </Container>
  );
}
