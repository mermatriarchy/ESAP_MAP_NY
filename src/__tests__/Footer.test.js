import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../components/common/Footer';

it('correctly renders About component', () => {  
    const FooterComponent = renderer.create(<Footer />).toJSON();
    expect(FooterComponent).toMatchSnapshot();
});