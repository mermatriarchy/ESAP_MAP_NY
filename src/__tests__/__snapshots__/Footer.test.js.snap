// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders About component 1`] = `
<footer
  className="bg-dark container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-md-6"
    >
      <p
        className="text-white"
      >
        Copyright 2019-2020, H. Adonis Miller
        <br />
        <span
          className="font-weight-light"
        >
          This is a community-focused project not intended for use by police, military personnel or any adjacent third party affiliates or contractors; it is not intended to aid in surveillence of our communities in any capacity.
        </span>
      </p>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <dl>
        <dd>
          <a
            className="text-info"
            href=""
          >
            Source
          </a>
        </dd>
        <dd>
          <a
            className="text-info"
            href=""
          >
            Report a technical issue
          </a>
        </dd>
      </dl>
    </div>
  </div>
</footer>
`;
